README.md

```
@@ -249,13 +249,15 @@ through `127.0.0.1:8000` which is exposed by the Kind cluster.
 For the tests to work correctly, the following host names must resolve to `127.0.0.1`:
 
 - `keycloak.keycloak.svc.cluster.local` - The Keycloak identity provider used for authentication.
-- `fulfillment-api.osac.svc.cluster.local` - The fulfillment service API.
+- `fulfillment-api.osac.svc.cluster.local` - The fulfillment service external API.
+- `fulfillment-internal-api.osac.svc.cluster.local` - The fulfillment service internal API.
 
 Add the following entries to your `/etc/hosts` file:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)